QRCoder.Xaml/Properties/AssemblyInfo.cs

```diff
@@ -1,35 +0,0 @@
-using System.Reflection;
-using System.Runtime.InteropServices;
-
-// Allgemeine Informationen über eine Assembly werden über die folgenden 
-// Attribute gesteuert. Ändern Sie diese Attributwerte, um die Informationen zu ändern,
-// die mit einer Assembly verknüpft sind.
-[assembly: AssemblyTitle("QRCoder")]
-[assembly: AssemblyDescription("QRCoder is a simple library, written in C#.NET, which enables you to create QR Codes. It's licensed under the MIT-license.")]
-[assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("www.code-bude.net")]
-[assembly: AssemblyProduct("QRCoder")]
-[assembly: AssemblyCopyright("Copyright © www.code-bude.net / Raffael Herrmann.  All rights reserved.")]
-[assembly: AssemblyTrademark("written by Raffael Herrmann")]
-[assembly: AssemblyCulture("")]
-
-// Durch Festlegen von ComVisible auf "false" werden die Typen in dieser Assembly unsichtbar 
-// für COM-Komponenten. Wenn Sie auf einen Typ in dieser Assembly von 
-// COM zugreifen müssen, legen Sie das ComVisible-Attribut für diesen Typ auf "true" fest.
-[assembly: ComVisible(true)]
-
-// Die folgende GUID bestimmt die ID der Typbibliothek, wenn dieses Projekt für COM verfügbar gemacht wird
-[assembly: Guid("e668b98b-83bb-4e60-b33c-4fd5ed9c0156")]
```

Comment:
The old GUID for COM was incorrectly set the same as the QRCoder's GUID.  A new GUID has been assigned for QRCoder.Xaml.